Task6/Book.cs

```diff
@@ -0,0 +1,27 @@
+﻿
+namespace Task6
+{
+    public class Book
+    {
+        public string Title { get; }
+        public DateTime? PublicationDate { get; }
```

Comment:
Authors has to be objects of class Author (first name, last name, date of birth, limitation for names- not more then 200 symbols) of the book (collection can be empty). Please fix